### DISCUSSION TITLE
Equivalent Method to DOBlendableLocalMoveBy in PrimeTween

### DISCUSSION THREAD

**Discussion by dpshah1234** [KyryloKuzyk/PrimeTween#55] [FAQ] [answered]
We're looking for a method within PrimeTween that functions similarly to DOTween's DOBlendableLocalMoveBy. Our goal is to perform a blendable local movement animation on an object, allowing for smooth transitions and adjustments during the animation. If PrimeTween offers an equivalent or a workaround to achieve similar effects, guidance on how to implement it would be greatly appreciated.

**Comment by KyryloKuzyk** [comment, marked as answer]
Hey @dpshah1234,

PrimeTween has 'blendable' tweens, but they are called 'additive'. Additive tweens are currently experimental because of this shortcoming (DOTween also has it):
> But there is also one small caveat: 'additive' tweens accumulate floating point error over time. This error accumulation may not be a problem for rare animations, but if you repeat an additive tween thousands of times, the accumulated error may be significant. This is why this feature is experimental.

More info:
https://forum.unity.com/threads/primetween-high-performance-animations-and-sequences.1479609/#post-9241497
https://forum.unity.com/threads/primetween-high-performance-animations-and-sequences.1479609/page-4#post-9540721

There is no way to fix the error accumulation and I don't want to release a feature that breaks in some circumstances.

**Comment by coty-crg** [comment]
Please make it official. I followed the instructions and enabled it. I understand the risks but in our case it does not matter (we reset back to zero, then use blendables for a short period). Just note the risks in the summary of the function and call it a day.

**Reply by KyryloKuzyk** [reply]
I’m still considering it. I don’t want to rush with such decisions because they are final, meaning that once the feature is there, there is no way back. And if a feature has vulnerabilities, it’s better be safe than sorry. 

Describing these limitations in the function’s docs is good idea, although in my experience people rarely read documentation. And when something will not work as expected, people will blame the tool (which is reasonable). So having this feature guarded under the experimental flag protects users from shooting themselves in a foot.
